portals/publisher/src/main/webapp/source/src/app/components/Apis/Create/Components/DefaultAPIForm.jsx

```diff
@@ -485,6 +515,22 @@ export default function DefaultAPIForm(props) {
                     margin='normal'
                     variant='outlined'
                 />
+                <TextField
+                    fullWidth
+                    id='api-display-name'
+                    label={(
+                        <>
+                            <FormattedMessage id='Apis.Create.Components.DefaultAPIForm.display.name'
+                                defaultMessage='Display Name' />
+                        </>
```

Comment:
Do we need to wrap this formattedMessage?

```suggestion
                        <FormattedMessage
                           id='Apis.Create.Components.DefaultAPIForm.display.name'
                           defaultMessage='Display Name'
                        />
```